topics/dev/tutorials/conda/slides.html

```diff
@@ -19,13 +19,11 @@
 time_estimation: "75m"
 requirements:
   -
-    title: "Tool development and integration into Galaxy"
     type: "internal"
-    link: "/topics/dev/tutorials/tool-integration/slides.html"
-  -
-    title: "Prerequisites for building software/conda packages"
-    type: "internal"
-    link: "/topics/dev/tutorials/conda_sys/slides.html"
+    topic_name: dev
+    tutorials:
+      - tool-integration
+      - conda_sys
```

Comment:
@bebatut Both of these requirements exist only as slide decks (not tutorials), so these are now broken links, see https://travis-ci.org/galaxyproject/training-material/jobs/505792486#L2098

Reply:
I found out the error: there is hands-on tutorial but with external link. It should be fixed with #1317

Reply:
Thanks for looking at this @bebatut !